### PR TITLE
Feature/serializer updates

### PR DESCRIPTION
**Pull request by z1haze**
Overview:
---

Updates to how we serialize arrays and array-like objects

Reviewer:
---

> Where should the reviewer start? How to Test? Background Context? etc ( required )

Test serializer with object that contains arrays or List types

Checklist:
---

> I have tested each of the following, and they work as expected: ( required )

- [x] Meets [Contributing Guide](https://github.com/redvanworkshop/rvw_developers_core/blob/develop/.github/CONTRIBUTING.md) Requirements
- [x] Pulled in the Latest Code from the `develop` branch
- [x] Works on a Desktop / Laptop Device
- [ ] Works on a Mobile Device
- [ ] `npm test` Does Not Generate Errors

Documentation:
---

> Screenshots, Attachments, Linked GitHub Issues, etc ( optional )

#### What GIF best describes this PR or how it makes you feel?

> Drag & Drop Something Fun Here ( optional )

### CODE REVIEW

**Comment by manifestinteractive** [comment]
@z1haze can you also update the package-lock.json with a matching version number ?

Sorry about the CLA thing, it was supposed to white list you, but looks like that did not work.  I can manually push this through once the lock file is updates.  Everything else looked good.

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "rvw_developers_core",
-  "version": "1.1.0",
+  "version": "1.1.1",
```
Comment (manifestinteractive) [inline package.json:3]: Can you also update the package-lock.json file ?

**Review by manifestinteractive — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (z1haze) [reply package.json:3]: what in the lock file needs updated?

Comment (z1haze) [reply package.json:3]: done.

**Review by manifestinteractive — APPROVED** [review]

**Pull request merged**